deploy/crd.yaml

```diff
@@ -4,6 +4,8 @@ metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.16.3
   name: perconaservermysqlbackups.ps.percona.com
+  labels:
+    mysql.percona.com/version: v0.10.0
```

Comment:
@pooknull we need to do it not only for perconaservermysqlbackups but for all our CRDs

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/850/commits/3fb7a0498207b6f39ada4213e804a043a8300787